lib/src/chewie_controller.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:chewie/src/chewie_progress_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:video_player/video_player.dart';
+
+class ChewieController extends ChangeNotifier {
```

Comment:
Can we please add some DartDocs (e.g. `/// Triple slash coments`) to this, even if it's just a line or two?

Reply:
Sure, can you come up with a short and precise text for it? I tried but postponed it..

Reply:
```
/// The ChewieController is used to configure and drive the Chewie Player Widgets. It
/// provides methods to control playback, such as [pause] and [play], as well as methods
/// that control the visual appearance of the player, such as [enterFullScreen] or 
/// [exitFullScreen].
///
/// In addition, you can listen to the ChewieController for presentational changes, such as
/// entering and exiting full screen mode. To listen for changes to the playback, such as 
/// a change to the seek position of the player, please use the standard information provided 
/// by the `VideoPlayerController`.
```

How does that sound?

Reply:
Sounds good to me. Thanks!